### ISSUE TITLE
Sort starts-with matches to top of list.

### ISSUE THREAD

**Issue by SamHasler** [jakiestfu/Mention.js#2]

**Comment by jakiestfu** [comment]
@SamHasler What do you mean? Results are sorted by highest match.

**Comment by SamHasler** [comment]
If I type "@C" then Charlie should be at the top of the list.

**Comment by jakiestfu** [comment]
I see how that can be deemed an issue, but after much thought, I'm deciding to leave it the way it is. The nature of other services like Twitter, Facebook, and Instagram all allow matching of characters that don't necessarily start with the specified character, which I feel is very useful

**Comment by SamHasler** [comment]
I don't think I've explained very well. Let me try again. I'm not saying
only show starts-with matches. I'm saying do both, but put the matches with
the search term at the start at the top of the list.

E.g. after typing  "@c" the list might be :

<b>C</b>harlie,
<b>C</b>onlin
Miss<b>C</b>J
YoungRe<b>c</b>kless
Ri<b>c</b>kyBahner
Ha<b>c</b>kMurphy
NikRi<b>c</b>hie

This puts the names that the user is more likely to be typing first.

Note also how MissCJ is next. You could sort matches with capitals after
lowercase above other matches as they are also more likely what the user
wants.

**Comment by jakiestfu** [comment]
@SamHasler I have updated the script and the documentation to reflect the new "sensitivity" feature that sorts results based off of various conditions.

**Comment by SamHasler** [comment]
Sweet!
